centraldogma/src/main/java/com/linecorp/decaton/centraldogma/CentralDogmaPropertySupplier.java

```diff
@@ -141,23 +144,51 @@ public void close() {
      */
     public static CentralDogmaPropertySupplier register(CentralDogma centralDogma, String project,
                                                         String repository, String filename) {
-        createPropertyFile(centralDogma, project, repository, filename);
+        createPropertyFile(centralDogma, project, repository, filename, ProcessorProperties.defaultProperties());
+        return new CentralDogmaPropertySupplier(centralDogma, project, repository, filename);
+    }
+
+    /**
+     * Create a default property file if it doesn't exist on Central Dogma and
+     * return a {@link CentralDogmaPropertySupplier}.
+     * @param centralDogma a {@link CentralDogma} instance to use to access Central Dogma server.
+     * @param project the project name where the properties are placed.
+     * @param repository the repository name where the properties are placed.
+     * @param filename the name of the file containing properties as top-level fields.
+     * @param supplier user-customized settings for kafka-consumer.
```

Comment:
These properties are of decaton, not kafka-consumer. So let's reword it like: `a {@link PropertySupplier} which provides set of properties that wants to have a custom initial value`

Reply:
I reworded the javadoc for supplier: 
`a {@link PropertySupplier} which provides a set of properties with customized initial values`
https://github.com/line/decaton/pull/134/commits/2e70198681ebd07786d1e4e69004364e51b31f30